static/keys/public.key

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
I'd like to see this file encrypted with the sandbox GPG key please

Reply:
This is now addressed.